src/Icons/Icon.tsx

```diff
@@ -8,7 +8,7 @@ export {IconSVGs};
 
 const IconWrapper = styled.div<{color:string}>`
   [stroke]{
```

Comment:
This is pretty weird looks for elements with a attribute stroke and override with CSS. But I guess this is OK based on this.
 https://www.w3.org/TR/SVG/styling.html#PresentationAttributes